RadarSDK/RadarSettings.m

```diff
@@ -226,6 +229,22 @@ + (void)setFeatureSettings:(RadarFeatureSettings *)featureSettings {
     }
 }
 
++ (NSDictionary *)clientSdkConfiguration {
+    NSDictionary *sdkConfigurationDict = [[NSUserDefaults standardUserDefaults] dictionaryForKey:kClientSdkConfiguration];
+    if (sdkConfigurationDict == nil) {
+        sdkConfigurationDict = [[NSDictionary alloc] init];
+    } 
+    return sdkConfigurationDict;
+}
+
++ (void)setSdkConfiguration:(RadarSdkConfiguration *)sdkConfiguration {
+    [[RadarLogger sharedInstance] logWithLevel:RadarLogLevelDebug
+        message:[NSString stringWithFormat:@"Setting SDK configuration | sdkConfiguration = %@",
+                            [RadarUtils dictionaryToJson:[sdkConfiguration dictionaryValue]]]];
+    if (sdkConfiguration) {
+        [[NSUserDefaults standardUserDefaults] setInteger:(int)sdkConfiguration.logLevel forKey:kLogLevel];
```

Comment:
why are we storing just an int here? Why not the whole `RadarSdkConfiguration` obj